### ISSUE TITLE
[Other] Support for document versioning?

### ISSUE THREAD

**Issue by AlexKalopsia** [jonaswinkler/paperless-ng#685]
Sometimes I have documents such as contracts, and I'd like to keep track of any possible update of the contract itself without creating separate entities. 

Is document versioning supported in any way?

**Comment by jonaswinkler** [comment]
No, not supported. If you really need that, I'd look elsewhere.

**Comment by AlexKalopsia** [comment]
Honestly, I've looked elsewhere and this is by far the best document manager, so I can live without. But it still feels like there might be some "cheap" ways to support this? I saw a few people suggested linking documents. An alternative could be to use versioning within the document serial? so if the ID is 11, I could then do 11.1 11.2 11.3 and so on? Just food for thought, it doesn't have to be a complex versioning system. 

But yeah, again, it's not the end of the world :)

**Comment by w0ndersp00n** [comment]
You might take a look at https://www.mayan-edms.com/
It’s quite heavy and more geared to enterprises, but does support versioning if that’s a requirement. I personally don’t use it due to it’s high system requirements and many features i don’t use.

**Comment by AlexKalopsia** [comment]
Thanks! Again, here I was not looking for something complex, but just a way to "group" documents together (not tags)

**Comment by fignew** [comment]
Perhaps #56 could meet your needs?
